lib/assessment.js

```diff
@@ -8,15 +8,24 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue, index, array) { previousValue = array.slice();
+     array.forEach(function(currentValue, index, array){
+      if (previousValue > currentValue){
+        previousValue = currentValue;
+      }
+     })
+     return previousValue;
```

Comment:
The question asks for a callback function, min, which we can pass into reduce, i.e. [1,2,3].reduce(min);

Here's an example of one we might have used.
```javascript
var min = function(previous, current) {
  if (previous < current) {
    return previous;
  } else {
    return current;
  }
}
```